src/DGtal/topology/doc/moduleCubicalComplex.dox

```diff
@@ -0,0 +1,328 @@
+
+/* 
+ * Useful to avoid writing DGtal:: in front of every class.
+ */
+namespace DGtal {
+
+/**
+   
+@page moduleCubicalComplex  Cubical Complex
+
+@writers Jacques-Olivier Lachaud
+
+@since 0.9.1
+
+Part of the \ref packageTopology.
+  
+This part of the manual describes how to represent and process
+arbitrary cubical complexes. 
+
+@note Collapse operation is a backport from <a
+href="https://gforge.liris.cnrs.fr/projects/imagene">ImaGene</a>. \cite
+ImaGene 
+
+[TOC]
+
+
+The following programs are related to this documentation:
+cubical-complex-collapse.cpp, cubical-complex-illustrations.cpp,
+testCubicalComplex.cpp,
+
+@section dgtal_ccomplex_sec1  Introduction to cubical complexes
+
+We define a \b cubical \b complex \a C as a collection of cells living
+in some Khalimsky space. Two cells of \a C are \b incident if and only
+if they are incident in the Khalimsky space. 
```

Comment:
maybe you should describe what new services/features are provided by the CubicalComplex compared to Khalimsky.